openzwavemqtt/options.py

```diff
@@ -12,11 +12,13 @@ def __init__(
         self,
         send_message: Callable[[str, Union[str, dict]], None],
         topic_prefix: str = "OpenZWave/",
+        instance_id: Optional[str] = None,
```

Comment:
Should we make this an integer instead of a string? The item id of the instance is an integer and this represents the same thing. We can copy it to a string before we filter.

https://github.com/cgarwood/python-openzwave-mqtt/blob/4a1ee92c81cca3d7c8df11ce5636e6f5a9801acd/openzwavemqtt/base.py#L110

Reply:
I changed the instance id to an integer since the id must be an integer. It's just when parsing the topic string that it becomes a string.